Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:alpine
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN adduser -S sopel
+
+COPY requirements.txt /usr/src/app/
```

Comment:
his can be changed to `COPY . /usr/src/app` since `requirements.txt` is already in `.`. Will save on image size (a super tiny amount, but one less layer)